Readme.md

```diff
@@ -0,0 +1,61 @@
+# userChrome Loader
+This is a template intended to make it easier for end-users to manually load and manage custom CSS.
+
+## Setup Instructions
+**Step 1.** In your **chrome** folder, save the **userchrome-loader** folder from this repo.
```

Comment:
Might be better to use standard numbered lists here so we can take advantage of the numbered list markdown.

e.g.
1. do this
2. do that